include/verilated.mk.in

```diff
@@ -202,19 +197,15 @@ $(VM_PREFIX)__ALL.a: $(VK_OBJS)
 ### Compile rules
 
 ifneq ($(VM_DEFAULT_RULES),0)
-$(VM_PREFIX)__ALLfast.o: $(VM_PREFIX)__ALLfast.cpp
+$(VM_PREFIX)__ALL.o: $(VM_PREFIX)__ALL.cpp
 	$(OBJCACHE) $(CXX) $(CXXFLAGS) $(CPPFLAGS) $(OPT_FAST) -c -o $@ $<
 
-$(VM_PREFIX)__ALLslow.o: $(VM_PREFIX)__ALLslow.cpp
-	$(OBJCACHE) $(CXX) $(CXXFLAGS) $(CPPFLAGS) $(OPT_SLOW) -c -o $@ $<
-
-# VM_GLOBAL_FAST files including verilated.o use this rule
+# Anything not in $(VK_SLOW_OBJS), including verilated.o use this rule
 %.o: %.cpp
 	$(OBJCACHE) $(CXX) $(CXXFLAGS) $(CPPFLAGS) $(OPT_FAST) -c -o $@ $<
 
-%__Slow.o: %__Slow.cpp
+$(VK_SLOW_OBJS): %.o: %.cpp
```

Comment:
If someone has any user files named __Slow this will change behavior.  I also sort of like having a name that I can be sure compiles a certain way.  Quite open to counter-argument.

Reply:
I did this because __Syms.cpp is a slow file but the old rule doesn't aply to that.

Options are:
1 Don't care, keep building __Syms.cpp as fast.
2 Emit as __Syms__Slow.cpp 
3 Keep new rule but add old rule as well for backwards compatibility
4 Keep old rule, add new rule just for __Syms.cpp (will break if we add another slow file that is not *__Slow.cpp)

I appreciate backwards compatibility but at the same time, I don't think this is a documented feature, so in the best case people are just piggy backing on it. I personally don't like filenames having any meaning but a human readable name as I never remember what substring is magic in who's build system+ also might have to do with me not being able to type, though that is not much of an argument :)

I also did it this way because I was planning to add a 3rd rule later with OPT_GLOBAL (based on what we discussed in #2247) to use for the run-time library bits, as for a speedy debug cyle one might want OPT_FAST=-O0, but the run-time lib is tiny in comparison and contains all tracing routines so OPT_FAST=-O0 and OPT_GLOBAL=-O2 would be way faster with tracing at insignificant build time cost (especially that the GLOBAL files ccache well). That new rule would look the same but with `s/SLOW/GLOBAL/g`.

Reply:
Ok with how you did it now that I understand, thanks.